### PR TITLE
Hometask 5

### PR DESCRIPTION
**Pull request by Hiblton**
React hooks

### CODE REVIEW

File: src/components/MovieDetails/MovieDetails.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          src={movie?.movieUrl}
+          alt={movie?.title}
+        />
+        <div className={styles.description}>
+          <div className={styles.inlineFlex}>
+            <Title title={movie?.title}></Title>
+            <div className={styles.rating}>7.3</div>
+          </div>
+          <span className={styles.movieGenre}>{movie?.genre}</span>
+          <div className={`${styles.inlineFlex} ${styles.dateAndRintime}`}>
```
Comment (i9or) [inline src/components/MovieDetails/MovieDetails.js:20]: > dateAndRintime

Typo in `Runtime` word.

**Review by i9or — COMMENTED** [review]
LGTM

PS Sorry for the late review, was unavailable.

File: src/layout/Header/Header.js [hunk 2]
```diff
@@ -18,17 +21,30 @@ const Header = () => {
...
+            <button
+              className={`${styles.button} ${styles.transparent}`}
+              onClick={() => setAddMovieModalOpened(true)}
+            >
+              + ADD MOVIE
+            </button>
+          )}
         </div>
+        {selectedMovie ? (
+          <MovieDetails movie={selectedMovie}></MovieDetails>
```
Comment (i9or) [inline src/layout/Header/Header.js:41]: Could be also written as

```jsx
<MovieDetails movie={selectedMovie} />
```

**Comment by i9or** [comment]
Oh, actually I haven't seen any `useEffect` in the project. As I remember at this point you should have already used fake backend service, available in the task. Chat me, if you need guidance.

**Comment by Hiblton** [comment]
> Oh, actually I haven't seen any `useEffect` in the project. As I remember at this point you should have already used fake backend service, available in the task. Chat me, if you need guidance.
> 
> **Edit:** checked the task again, you need to start using mock back-end with the next task. So `useEffect` _could be_ used to change the web document `title` for example. But actually that does not make much sense. Let's discuss it on the Q&A call.

I used 'useEffect' with modal windows and Portal in prev hometask. https://github.com/Hiblton/react-mentoring/blob/131a60726a5aa23aa1cb56545f020dc04d509231/src/layout/Portal/Portal.js#L10

**Comment by Hiblton** [comment]
Thanks for the review!

**Comment by i9or** [comment]
> I used 'useEffect' with modal windows and Portal in prev hometask. 

Ah, my bad, forgot about it. No issues then. The task is done.

**Pull request merged**